ui-meta/multinode-infra-appliance.yml

```
@@ -0,0 +1,183 @@
+    # Defaults to false if not given.
+    # immutable: true
+    # Additional options for the parameter. The valid options depend on the kind of the parameter.
+    #   list
+    #     min_length: An optional minimum length for the list.
+    #     max_length: An optional maximum length for the list.
+    #     item: An optional item validator, consisting of a kind and some options.
+    #   string
+    #     min_length: An optional minimum length for the string.
+    #     max_length: An optional maximum length for the string.
+    #     pattern: An optional regular expression pattern to validate the string against.
+    #   integer
+    #   number
+    # options:
+    #   min: 1
+    #   max: 100000
+    #   boolean
+    #     permanent: If true, indicates that the parameter cannot become false again once it has
+    #                been set to true. Defaults to false if not given.
+    #   choice
+    #     choices: The list of valid choices for the parameter. This field is required.
+    #   cloud.size
+    #     min_cpus: An optional minimum number of CPUs that the size must have.
+    #     min_ram: An optional minimum amount of RAM in MB that the size must have.
+    #     min_disk: An optional minimum root disk size in GB that the size must have.
+    #   cloud.volume
+    #     min_size: An optional minimum size in GB for the volume.
+    #   cloud.cluster
+    #     tag: An optional tag that the cluster must have.
+    # options:
+    #  min_ram: 1024
+    #  min_disk: 10
+    #  secret: true
+    #  confirm: true
+  
+# A list of parameters to be gathered from the user
+parameters:
+  - name: vxlan_vni
+    label: Set VXLAN VNI.
+    description: Please provide a VXLAN VNI. A unique value from 1 to 100,000.
+    kind: "integer"
+    required: true
+    default: null
+    immutable: true
+    options:
+      min: 1
+      max: 100000
+      
+
+  - name: ans_vlt_pwd
+    # A human-readable name for the parameter. Defaults to the name if not given.
+    label: Ansible Vault password.
+    # An optional short description for the parameter.
+    description: Please provide the Ansible Vault password to enable the deployment of the cluster.
+    kind: "string"
+    required: false
+    #default: "password_not_set"
+    immutable: false
+    # Replace the characters of the parameter with asterisks in the UI.
+    options:  
+      secret: true
+      confirm: true
+  
+  - name: openstack_deploy 
+    # A boolean flag which will be set to true if the user selects the checkbox,
+    # this will allow the user to decide whether they wish to have the
+    # infrastructure built with or without OpenStack.
+    label: Deploy Infrastructure with OpenStack services? #(True/False)
+    description: If you wish to deploy OpenStack services in the infrastructure select this option.
+    kind: "boolean"
+    default: false
+    required: false
+    immutable: true
+
+  - name: kayobe_version_branch
```

I don't see this variable used anywhere.
```suggestion
  - name: kayobe_branch
```

---

There is a related issue here, which is that in deploy-openstack.sh we will install whatever version of kayobe is referenced in SKC's requirements.txt, so this input (and the ansible variable it maps to is not really being used).

It might be better to make a separate change to stop installing kayobe from source, and always install it from SKC requirements.txt.

---

> I don't see this variable used anywhere.

These variables are used in the full multinode deployment version of the App (`Multinode-Deployment-App`) and those changes are not included in this version due to relating to the deployment of OpenStack on the multinode, rather than just the infrastructure.

> There is a related issue here, which is that in deploy-openstack.sh we will install whatever version of kayobe is referenced in SKC's requirements.txt, so this input (and the ansible variable it maps to is not really being used).

If they aren't really being used, can we remove them all together in `/ansible/vars/defaults.yml`?